.github/workflows/release-binaries.yml

```diff
@@ -484,7 +343,17 @@ jobs:
       id: setup-stage
       uses: ./workflows/.github/workflows/release-binaries-setup-stage
       with:
-        previous-artifact: build-stage3-all
+        previous-artifact: build-release-package
+
+    # Need sccache installed, because some stage1 objects are being built for the tests.
+    # FIXME: This probably shouldn't be happening.
```

Comment:
Do you have a list of what is built? Is `stage2-packages` supposed to depend on the `-test-depends` targets?